src/auth/auth-models.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+
+/** set up auth user model **/
+const authSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
```

Comment:
I am going to disagree on this comment.

There is only one `password` variable. The only time a plaintext password is handled is when it comes from the client. The `password` in the database is hashed.

In my opinion, leaving the field name as `password` provides clarity on the intended use of the field as well as abstracting away from the security details.